crates/sui-types/src/error.rs

```diff
@@ -73,7 +74,7 @@ macro_rules! invariant_violation {
 
 #[macro_export]
 macro_rules! assert_invariant {
-    ($cond:expr, $($args:expr),*) => {{
+    ($cond:expr, $($args:expr),* $(,)?) => {{
```

Comment:
Looks like the `assert_invariant` call changed below doesn't have a trailing comma, so think this change isn't needed, but might be good practice anyway? Am not a macro expert :)

Reply:
Ah yes!  I'm not sure what combination of formatters and my own habits got me here, but you're right, this is not strictly necessary anymore, but is good practice anyway.